apps/api/src/app/events/usecases/send-message/test-send-email.usecase.ts

```diff
@@ -12,13 +15,15 @@ import { TestSendMessageCommand } from './send-message.command';
 import { ApiException } from '../../../shared/exceptions/api.exception';
 import { CompileEmailTemplate } from '../../../content-templates/usecases/compile-email-template/compile-email-template.usecase';
 import { CompileEmailTemplateCommand } from '../../../content-templates/usecases/compile-email-template/compile-email-template.command';
+import { GetNovuIntegration } from '../../../integrations/usecases/get-novu-integration';
 
 @Injectable()
 export class SendTestEmail {
```

Comment:
I am sorry for this but I had renamed the class and file name in a PR that just got merged yesterday. I can help you with the right changes if needed.

Reply:
I saw the conflicted file in the pipeline and I wrote that comment. Good wasn't too hard to fix!